### DISCUSSION TITLE
Pass data from your components to your layout

### DISCUSSION THREAD

**Discussion by rafael1delgado** [livewire/livewire#4704] [1. Help]
Hi all, how to pass a variable to extended layout in livewire render function. When I tried to pass a variable , it is not displaying in layout blade. Showing undefined variable error. What is the correct method to do that.

` 
return view('livewire.show-posts')->layout('layouts.base', ['title' => 'Show Posts'])
`

**Comment by joshhanley** [comment]
@rafael1delgado can you share what your `layouts.base` looks like?

Thanks!
